MesnardBarba-CiSE.md

```diff
@@ -252,12 +253,12 @@ This final story is about what happened when we went back to our _original_ code
 As we mentioned in the opening of this article, we adopted a set of practices years ago to make our research reproducible. 
 The study published as "Lift and wakes of flying snakes" was completed under the guidance of the "Reproducibility PI Manifesto," 
 which includes: 
-(1) the code was developed under version control; 
-(2) we completed validation and verification, publishing the report on Figshare; 
-(3) the data and figures for the main results of the paper are open; 
-(4) the pre-print is available on arXiv; 
-(5) the code was released under MIT License; 
-(6) we included a Reproducibility statement in the paper. 
+(1) code developed under version control; 
+(2) completed validation and verification, report published on Figshare; 
+(3) data and figures for the main results of the paper are open; 
+(4) pre-print is available on arXiv; 
+(5) code released under MIT License; 
+(6) include a Reproducibility statement in the paper. 
```

Comment:
Actually, scratch this "include", should just read
`(6) a Reproducibility statement...`